src/beast/beast/nudb/tests/recover_test.cpp

```diff
@@ -134,8 +134,10 @@ class recover_test : public basic_recover_test
         float lf = 0.75f;
         test_recover (lf, 0);
         test_recover (lf, 10);
+    #ifdef NDEBUG
```

Comment:
So that the test doesn't take too long. Release versions can process more items in the same time.